.pre-commit-config.yaml

```diff
@@ -1,12 +1,27 @@
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v3.2.0
-  hooks:
-  - id: trailing-whitespace
-  - id: end-of-file-fixer
-  - id: check-yaml
-  - id: check-added-large-files
-- repo: https://github.com/psf/black
-  rev: 23.7.0
-  hooks:
-    - id: black
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-added-large-files
+
+  - repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+      - id: black
+        name: "Black: The uncompromising Python code formatter"
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.12.0
+    hooks:
+      - id: isort
+        name: "Sort Imports"
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.1.13
```

Comment:
This version is different from our pyproject.toml — please sync them?

Reply:
https://github.com/qdrant/fastembed/commit/47ee7ad84cbd7e96774b22180d889c7fe1e434b2

Reply:
https://github.com/qdrant/fastembed/commit/5b66b70eb12c2dc14e295c3d92920592d5daf327

Reply:
This is still out of sync: https://github.com/qdrant/fastembed/blob/b01f882df7113b24b37391a8ada65ba5e4886bb7/pyproject.toml#L24